code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -90,6 +93,60 @@
 		user.filters = null
 		animation_playing = FALSE
 
+/obj/item/borg_chameleon/proc/fulp_borg_chameleon_menu(mob/living/silicon/robot/user) //Surrealaser's code
+	if(!user)
+		return FALSE
+
+	var/list/borg_disguses_list = list(
+	"Clown", \
+	"Engineering", \
+	"Janitor", \
+	"Medical", \
+	"Mining", \
+	"Peacekeeper", \
+	"Security", \
+	"Service", \
+	"Standard", \
+	"Cancel" = null )
```

Comment:
Make this a global list.
Else it will build it up every time the proc is called.